shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -164,4 +164,9 @@ FLUTTER_DARWIN_EXPORT
 - (instancetype)initWithOwner:(FlutterTextInputPlugin*)textInputPlugin NS_DESIGNATED_INITIALIZER;
 
 @end
+
+@interface UIView (FindFirstResponder)
+- (id)firstResponder;
```

Comment:
Can you rename to `flt_firstResponder` in case of naming conflict? (ObjC doesn't support namespace)

Reply:
Good idea! Went through and made the change.